### DISCUSSION TITLE
RTL8821CU Wi-Fi working

### DISCUSSION THREAD

**Discussion by antergosnm** [chris1111/Wireless-USB-Big-Sur-Adapter#315] [Show and tell]
Hi, just wanted to mention that this WiFi card is working in macOS Ventura with SIP enabled and notarized driver. (and no, it's not a USB WiFi dongle, is my internal WiFi card).

- iServices working
- Connection is stable

**Comment by chris1111** [comment]
Thanks @antergosnm 
What is the brand and name of the card?
What is the PC, Laptop?

**Reply by antergosnm** [reply]
It's a generic card soldered on the board. The chip is a Realtek RTL8821CU. Perhaps it's connected on a USB-like bus. Bluetooth doesn't work but WiFi does, and it's stable. And it's a laptop.

**Reply by JuKaldosh** [reply]
I have a RTL8821BE on a Z370e an can't wait it to work natively , would be such a waste even if cheap solution not to be able to
